src/widgets/buttons.ts

```diff
@@ -477,24 +488,62 @@ export function deleteButtonWithCheck (
   return deleteButtonElt
 }
 
+function getButtonStyle (options: ButtonWidgetOptions) {
+  const color = '#7c4dff'
+
+  const backgroundColor = options.filled ? color : '#ffffff'
```

Comment:
this is just the beginning I'm going to make this better by adding a case statement and also need to look deeper into the style so that it is the same as the SDK

Reply:
Actually maybe even separate functions for each buttonType....  but I am not sure if that's going too far.